src/main/kotlin/racingcar/RacingGame.kt

```diff
@@ -1,18 +1,32 @@
 package racingcar
 
-class RacingGame(private val carCount: Int, private val tryCount: Int) {
-    val driverList = List(this.carCount) { Driver(Car()) }
+class RacingGame(val drivers: List<Driver>, val tryCount: Int) {
+    private val carCount = drivers.size
 
     fun start() {
         repeat(tryCount) {
-            driverList.forEach {
+            drivers.forEach {
                 val isMovable = MOVE_RANGE.random() >= MOVE_CONDITION
                 it.drive(isMovable)
             }
         }
     }
 
+    fun getWinner(): List<String> {
+        val maxMovement = drivers.maxOf { it.driveRecord.last() }
+
+        return drivers.filter { it.driveRecord.last() == maxMovement }.map { it.name }
```

Comment:
함수를 연속해서 호출할때의 [컨벤션](https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls)입니다~! 
```suggestion
        return drivers.filter { it.driveRecord.last() == maxMovement }
            .map { it.name }
```